server/queues/scoring/createNewTask.coffee

```diff
@@ -26,7 +26,7 @@ module.exports = createNewTask = (req, res) ->
 
 
 validatePermissions = (req, sessionID, callback) ->
-  return callback 'You are unauthorized to submit that game to the simulator.' unless req.user?.get('email')
+  return callback 'You are unauthorized to submit that game to the simulator.' if not req.user or req.user.isAnonymous()
```

Comment:
I'd put parens around `not req.user` to be clear what the `not` is referring to. Makes it easier for me to grok, anyway.